frontend/src/App.jsx

```diff
@@ -17,7 +17,7 @@ import { socket } from "./socket";
 import { useEffect } from "react";
 import { BASE_URL } from "./lib/utils";
 
-const LoggedInPage = ({ isLoggedIn, children }) => {
+const LoggedInPage = ({ isLoggedIn, children, isGuest }) => {
```

Comment:
I'm not completely following why this needs to be a new param for `LoggedInPage`. Isn't being a guest, for all intents and purposes, the same as being logged in, at least from the perspective of this wrapper?